Doc/whatsnew/3.10.rst

```diff
@@ -1691,6 +1691,13 @@ New Features
   singleton or the ``False`` singleton.
   (Contributed by Victor Stinner in :issue:`43753`.)
 
+* Add new functions to quickly control the garbage collector from C code:
+  :c:func:`PyGC_Enable()`,
+  :c:func:`PyGC_Disable()`,
+  :c:func:`PyGC_IsEnabled()`.
```

Comment:
Please document these functions but also PyGC_Collect().

https://docs.python.org/dev/c-api/gcsupport.html may be good home for this.

I'm disappointed, PyGC_Collect() is not documented at all!

Reply:
Yeah, I was kind of considering that they might have been intentionally undocumented. But probably not. I'll write something up.

Reply:
Done.